specification/securitydevops/resource-manager/Microsoft.SecurityDevOps/preview/2022-09-01-preview/securitydevops.json

```diff
@@ -0,0 +1,2692 @@
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AzureDevOpsConnectorName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AzureDevOpsRepoListResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "AzureDevOpsRepo_ListByConnector": {
+            "$ref": "./examples/AzureDevOpsRepoListByConnector.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SecurityDevOps/azureDevOpsConnectors/{azureDevOpsConnectorName}/stats": {
+      "get": {
+        "tags": [
+          "default"
+        ],
+        "summary": "Returns the summary of the AzureDevOps Connector Stats.",
+        "operationId": "AzureDevOpsConnectorStats_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AzureDevOpsConnectorName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AzureDevOpsConnectorStatsListResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "AzureDevOpsConnectorStats_Get": {
+            "$ref": "./examples/AzureDevOpsConnectorStatsGet.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SecurityDevOps/azureDevOpsConnectors/{azureDevOpsConnectorName}/orgs": {
```

Comment:
>azureDevOpsConnectors [](http://example.com/codeflow?start=107&length=21)

Naming thoughts - how reviewed is this resource name at this point? This might be late feedback but worries that occur to me are - should it be a product brand name like azureDevOps be a resource type name? What if it gets rebranded later? Is 'Azure' redundant in the context of Azure REST APIs? Is devOps implied in the context of SecurityDevOps?
Would just 'devOpsConnectors'? or 'connectors' have been as good?

Reply:
Hm. I guess you want a clear contrast with githubConnectors, so maybe this isn't as easy as I'd like.

Reply:
Since the type definitions look similar at a high level, I'm wondering if you also considered the option of discriminated types in the REST API.

Reply:
This already went through ARM API review once in June and I didnt find an easy way to just publish one api version from `azure-rest-apis-spec-pr` repo.

This PR is just our attempt to release already reviewed spec to public.